contrib/python-configobj/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "python-configobj"
+pkgver = "5.0.8"
+pkgrel = 0
+build_style = "python_module"
+hostmakedepends = ["python-setuptools"]
+depends = ["python-six"]
+pkgdesc = "Simple but powerful config file reader and writer"
+maintainer = "yopito <pierre.bourgin@free.fr>"
+license = "BSD-3-Clause"
+url = "https://github.com/DiffSK/configobj"
+source = f"https://github.com/DiffSK/configobj/archive/v{pkgver}.tar.gz"
+sha256 = "547dc047e31c71d7a8732016336769ed450588f34a7c13077aa7acc7df245eda"
+hardening = ["vis"]
```

Comment:
drop this, it's a python module without compiled parts

Reply:
my bad, thanks for pointing this